pkg/cmd/cli/cmd/expose.go

```diff
@@ -56,7 +56,13 @@ func NewCmdExpose(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.C
 		cmdutil.CheckErr(err)
 		defRun(cmd, args)
 	}
-	cmd.Flags().String("hostname", "", "Set a hostname for the new route")
+	cmd.Flags().String("hostname", "", "Set a hostname for the new route. Optional.")
+	cmd.Flags().String("path", "", "Path that the router watches to route traffic to the service. Optional.")
+	cmd.Flags().String("tls-termination", "", "Indicate a termination type for the generated route. If not set, any termination config will be ignored. Supports 'edge', 'passthrough', or 'reencrypt'.")
```

Comment:
> It is the default. If a user doesn't specify --tls-termination, the route will be unsecured.

@kargakis This part is the problem, "If not set, any termination config will be ignored.".  Why would I ever want to set a termination config and not use it?

It looks like the problem goes back to the API here: https://github.com/openshift/origin/blob/master/pkg/route/api/v1/types.go#L83.  I understand why the config is unusable without a `termination`, but I also don't see a reason to allow someone to create a useless route.  We should probably start with a separate pull there that makes `Termination` required (not `omitempty`) and tighten validation to require it if a `tlsConfig` is present.  Then bubbling out to here, we can avoid doing crazy things.